README.md

```diff
@@ -64,3 +64,14 @@ $ yarn build
 ```
 
 Built assets are written to the `build` directory. Point a static file server like [serve](https://github.com/vercel/serve) to that directory and you're good to go.
+
+## Docker 
+
+In order to use this project inside a docker container, just do:
+
+```zsh
+docker build -t react-scavenger-quiz .
+docker run -d -p 3000:80 react-scavenger-quit
+```
+
+Now you can access the quiz at http://&lt;ip-of-docker-server&gt;:3000/
```

Comment:
Would this ever not be `localhost`? Forgive me, not too familiar with Docker, but I was expecting to have to find an IP somewhere and was surprised to see it was just localhost.

Can we also add a note on how to stop the container?

```markdown
You can stop the container by running `docker stop <container ID returned by 'docker run'>`.
```

Reply:
It depents on which pc you are running and accessing it from. 
In my case the docker container is running on a publicly accessable server www.mydomain.com (ip: 1.2.3.4)

So in my case i can access it from "www.mydomain.com:3000" or "http://1.2.3.4:3000"

It is only localhost/127.0.0.1, if your browser and the docker container are running on the same machine.